File: algorandsamples/v2/java-test/src/main/java/com/algorand/javatest/smart_contracts/GetApplicationAddress.java
package com.algorand.javatest.smart_contracts;

import com.algorand.algosdk.crypto.Address;

public class GetApplicationAddress {
    public void getApplicationAddressExample() throws Exception {
        Long appId = 95223182L ;
        Address actual = Address.forApplication(appId);
        System.out.println("Address: " + actual.toString());
    }


public static void main(final String args[]) throws Exception {
    GetApplicationAddress t = new GetApplicationAddress();
    t.getApplicationAddressExample();}}

File: algorandsamples/v2/mygodemo/src/smart_contracts/getApplicationAddress.go
package main

import (
	"fmt"
	"github.com/algorand/go-algorand-sdk/crypto"
)


func main() {
	appID := 95223182
	actual := crypto.GetApplicationAddress(uint64 (appID))
	fmt.Printf("Application Address: %s\n", actual)
}